### PR TITLE
Fix placeholder selector case

### PR DESCRIPTION
**Pull request by xi**
> Which issue, if any, is this issue related to?

#2359

> Is there anything in the PR that needs further explanation?

No, it's self explanatory.

### CODE REVIEW

File: lib/utils/isStandardSyntaxTypeSelector.js [hunk 1]
```diff
@@ -31,5 +31,9 @@ module.exports = function (node/*: Object*/)/*: boolean*/ {
     return false
   }
 
+  if (node.value[0] === '%') {
```
Comment (evilebottnawi) [inline lib/utils/isStandardSyntaxTypeSelector.js]: Use `"` quotes

**Comment by xi** [comment]
Thanks for the quick reply! I adjusted the quotes.

**Comment by davidtheclark** [comment]
Great! Feel free to merge and add a changelog entry.

I just remembered that @hudochenkov suggested we release soon to get the Modernizr-conflict resolved. Should we do that after this merge?

**Comment by jeddy3** [comment]
Changelog

> -   Fixed: `selector-type-case` false positive for placeholder selectors ([#2360](https://github.com/stylelint/stylelint/pull/2360)).

**Pull request merged**